cmd/login/command.go

```diff
@@ -9,11 +9,36 @@ import (
 	"github.com/spf13/afero"
 	"github.com/spf13/cobra"
 	"k8s.io/client-go/tools/clientcmd"
+
+	"github.com/giantswarm/kubectl-gs/pkg/middleware"
+	"github.com/giantswarm/kubectl-gs/pkg/middleware/renewtoken"
 )
 
 const (
-	name        = "login"
-	description = "Logs in into the Kubernetes cluster."
+	name             = "login [K8s API URL | Web UI URL | Existing GS Context Name]"
+	shortDescription = "Logs into an installation's Kubernetes API"
+	longDescription  = `Log into an installation's Kubernetes API.
+
+You can use as an argument:
+  * Your installation's Kubernetes API URL as an argument (e. g. 'https://g8s.test.eu-west-1.aws.gigantic.io')
+  * Your Web UI URL (e. g. 'https://happa.g8s.test.eu-west-1.aws.gigantic.io')
+  * An existing Giant Swarm specific 'kubectl' context name (e. g. 'gs-test')
+`
+	examples = `  # See on which installation you're logged in currently.
+  kubectl gs login
+
+  # Log in using your K8s API URL.
+  kubectl gs login https://g8s.test.eu-west-1.aws.gigantic.io
```

Comment:
Again, please check whether this should be `api.g8s.`

Reply:
Isn’t this k8s api plugin? Why would it talk to giant swarm api?

Reply:
Right, it shouldn't talk to the rest API. It should also not talk to happa, but we are using the happa URL as a way for people to simply express "I want to use that control plane". But using the rest API was in fact not part of the spec. And that's fine. (I got confused about the spec)